src/Microsoft.Management.Configuration.Processor/Helpers/PowerShellGetV2.cs

```diff
@@ -17,6 +17,8 @@ namespace Microsoft.Management.Configuration.Processor.Helpers
     /// </summary>
     internal class PowerShellGetV2 : IPowerShellGet
     {
+        private const string AllUsers = "AllUsers";

```

Comment:
Wouldn't there actually be an enum type for this value? Or do they not expose those?

Reply:
noup, is just a validate set with two strings and even if they had an enum this is powershellget in psm1 files so no types in the sdk